src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int USER_OVER_MAX_AGE = 125;
+    private static final int USER_UNDER_MIN_AGE = 17;
+    private static User user;
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        StorageDao storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+        user.setId(3478700L);
+        user.setLogin("newUserLogin");
+        user.setPassword("userStrongPassword");
+        user.setAge(20);
+    }
+
+    @Test
+    void registerValidUser_Ok() {
```

Comment:
your tests method names should follow this pattern
`<methodUnderTest>_<state>_<expectedBehavior>`